main.go

```diff
@@ -23,7 +23,7 @@ import (
 	"time"
 
 	"github.com/spf13/pflag"
-	"go.uber.org/zap"
+	"go.uber.org/zap" // todo(camilamacedo86): why we are adding this lib? Shuld we not keep it == kube?
 	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
```

Comment:
why we are adding this lib? Shuld we not keep it == kube?

Reply:
How much more customizations we do now more complexity it will be. 
Also, the zap logs could be in the lube scaffold as well. i will track a task for that here in this repo for we know that is something that we need to do.